"thread_sampling_time_nsec": 4000000
"use_real_robot": false
"robot_driver_interface_node_prefixes": []
#Vrep Parameters
"vrep_port": 19998
"vrep_dynamically_enabled": true
"vrep_robot_joint_names": ["VS050_joint1#1","VS050_joint2#1","VS050_joint3#1","VS050_joint4#1","VS050_joint5#1","VS050_joint6#1"]

# This value is replaced by the enviroment variable VREP_IP
"vrep_ip": "0.0.0.0"
# This value is replaced by the enviroment variable ROBOT_1_JSON_PATH
"robot_parameter_file_path": "dummy.json"
